internal/resources/grafana/resource_api_key.go

```diff
@@ -146,13 +148,19 @@ func resourceAPIKeyDelete(ctx context.Context, d *schema.ResourceData, m interfa
 	return nil
 }
 
-func getClientForAPIKeyManagement(d *schema.ResourceData, m interface{}) (c *gapi.Client, cleanup func() error, err error) {
+func getClientForAPIKeyManagement(d *schema.ResourceData, m interface{}) (c *gapi.Client, cleanup func() error, err error, orgID int64) {
```

Comment:
I know this is a change with a massive blast radius, but can we keep the Goism of returning errors as the last returned value? 🙏🏽

Reply:
Yeah, of course. Dunno how I let that slip, that's a very weird func sig. Fixed!